x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/ml/text_expansion_search.yml

```diff
@@ -128,6 +128,32 @@ setup:
                   tokens_weight_threshold: 0.4
   - match: { hits.total.value: 4 }
   - match: { hits.hits.0._source.source_text: "the octopus comforter smells" }
+  - set: { no_boost_score: hits.hits.0._score }
```

Comment:
wow, wut?! You can set things from a different yaml runner and use it later!?!?

Reply:
Nope, that was an error that got pushed - difficulties testing locally. Fixing.